### ISSUE TITLE
Updating to 0.1.0 is throwing errors

### ISSUE THREAD

**Issue by NikhilPunwaney** [CraftyFella/winston-scalyr#14]
internal/modules/cjs/loader.js:311
      throw err;
      ^

Error: Cannot find module '/***/node_modules/winston-scalyr/build/scalyrTransport.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:303:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:516:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:867:27)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/***/logSetup.js:13:16)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/***/app.js:39:18)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10) {
  code: 'MODULE_NOT_FOUND',
  path: '/***/node_modules/winston-scalyr/package.json',
  requestPath: 'winston-scalyr'
}

**Comment by CraftyFella** [comment]
Thanks for this.. I'll double check in our app too

**Comment by CraftyFella** [comment]
i assume for now you can downgrade while I look into it?

**Comment by NikhilPunwaney** [comment]
Yes, I have downgraded for now. Looking forward to this fix as I am waiting on a feature that got merge into this from someone at Scalyr

**Comment by CraftyFella** [comment]
0.2.0 is now up.. and should be fixed.. i left out the build folder.

**Comment by CraftyFella** [comment]
let me know if you have any problems.. all looks good my end.. i got same error as you when pulling in the  0.1.0
